data/scripts/dependency-resolver.py

```diff
@@ -249,6 +249,12 @@ def handle_exec_check(args, conf, context):
     dep = conf.get("dependency")
     dep_sym = dep.upper()
     exe = conf.get("exec")
+    atleast_ver = conf.get("atleast-version")
+    max_ver = conf.get("max-version")
+    exact_ver = conf.get("exact-version")
+    cmd = conf.get("command-version")
```

Comment:
nitpick, but would `version-command` match better the usual names we're using here, like `exact-version`  instead of `version-exact`